docs/docusaurus/src/css/code_block.scss

```diff
@@ -0,0 +1,72 @@
+:root {
+  --code-snippets-title-color: #FAFAFA;
+  --code-snippets-title-background-color: #292F43;
+}
+
+div.theme-code-block {
+  box-shadow: 3px 3px 10px var(--black-15);
+  position: relative;
+
+  pre code {
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+    font-size: var(--p-md-font-size);
+  }
+
+  &:hover {
+    & > div:not(:only-child):first-child + div button {
+      opacity: 1;
+    }
+  }
+
+  // Code Block Title
+  & > div:not(:only-child):first-child {
+    background-color: var(--code-snippets-title-background-color);
+    box-shadow: 3px 3px 10px var(--black-15);
+    align-items: center;
+    border-bottom: none;
+    color: var(--code-snippets-title-color);
+    display: flex;
+    font-size: var(--p-md-font-size);
+    opacity: 0.75;
+    text-transform: capitalize;
+
+    &::before {
+      background: url('/img/file-solid.svg') 0 50% no-repeat;
+      content: '';
+      display: inline-block;
+      height: 1em;
+      margin-right: 0.5rem;
+      width: 1em;
+    }
+
+    // Code Block Content
+    & + div {
+      position: static;
+    }
+
+    // Button group
+    & + div > div {
```

Comment:
This feels a bit fragile, is there a way we can target this differently? Maybe with the classname instead?

Reply:
We don't have a classname to target and it's not possible to wrap the component to add them, the only alternatives I can think of are `& + div > div:has(button)` or `.prism-code > div` or `.prism-code > div:has(button)`. Let me know what you think or if you have any other suggestion!

Reply:
`.prism-code > div:has(button)` feels like a less brittle 👍🏽